client/packages.go

```diff
@@ -122,6 +123,16 @@ func (client *Client) List(names []string) ([]*Snap, error) {
 	return result, nil
 }
 
+// GetSections returns the list of existing snap sections in the store
+func (client *Client) GetSections() ([]string, error) {
+	var sections []string
+	_, err := client.doSync("GET", "/v2/sections", nil, nil, nil, &sections)
```

Comment:
(nitpick) This and the following line could be combined into a single: `if _err := client.doSync(..); err != nil { ` line. But its not really important (just an idiom we use a lot in the code).